.github/workflows/all-app-sandbox.yml

```
@@ -0,0 +1,102 @@
+  push:
+    branches:
+      - test
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    environment: all-app-sandbox
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)